students/KyleBarry/session02/fizzbuzz.py

```diff
@@ -0,0 +1,15 @@
+def fizz_buzz():
```

Comment:
good solid solution. If you want a bit of a challenge, see if you you refactor it to not compute % 3 and % 5 twice.